### PR TITLE
Create OSR-Roundtable.md

### PR DESCRIPTION
**Pull request by robmoffat**

### CODE REVIEW

**Comment by robmoffat** [comment]
Preview: https://deploy-preview-309--open-source-readiness.netlify.app/docs/bok/osr-resources/OSR-Roundtable-2024

File: docs/Presentations/OSR-Roundtable.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+- Secure Repository Access: GitProxy allows organizations to manage which external Git repositories their developers can access, ensuring that all code pulled into the organization meets security and compliance standards.
+- Code Vetting and Whitelisting: It can be configured to vet code from open-source repositories, automatically rejecting or flagging repositories that don’t meet certain security criteria or are from unknown sources.
+- Audit Trails and Compliance Logging: GitProxy provides detailed audit trails of repository access and code downloads, ensuring that all open- source usage is logged for compliance purposes. This feature is especially important for regulatory reporting and audit requirements. Integration with Internal Policies: It enforces organizational policies on open-source contributions, ensuring that only approved code is pushed to public repositories from the company’s developers
+
+# Reference Links:
+
+https://www.finos.org/blog/finos-q4-and-osff-announcements
```
Comment (psmulovics) [inline docs/Presentations/OSR-Roundtable.md:101]: I would suggest adding * to these two lines, as now they show up in the same line.

File: docs/Presentations/OSR-Roundtable.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+---
+title: OSR Roundtable at OSFF Oct 2024
+sidebar_label: OSFF-24
+sidebar_position: 10
```
Comment (psmulovics) [inline docs/Presentations/OSR-Roundtable.md:4]: Should we not position this the highest?

Comment (robmoffat) [reply docs/Presentations/OSR-Roundtable.md:4]: I've put them in the sidebar in the chronological order they happened.

Comment (psmulovics) [reply docs/Presentations/OSR-Roundtable.md:4]: Ah. Hoped for american chronological order :D (eg newest the top)

**Comment by tsteenbe** [comment]
One minor mistake search for 9SBOM), should be (SBOM)

**Review by psmulovics — APPROVED** [review]

**Pull request merged**